sdk/storage/storage-blob/package.json

```diff
@@ -1,17 +1,18 @@
 {
   "name": "@azure/storage-blob",
   "sdk-type": "client",
-  "version": "12.1.2",
+  "version": "12.2.0-preview.1",
   "description": "Microsoft Azure Storage SDK for JavaScript - Blob",
   "main": "./dist/index.js",
-  "module": "./dist-esm/src/index.js",
+  "module": "./dist-esm/storage-blob/src/index.js",
   "browser": {
```

Comment:
Sorry I forgot. Is this src => storage-blob change related to the common code approach?

Reply:
Yes.